pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/KitConfigurationDao.java

```diff
@@ -139,13 +140,14 @@ default KitConfiguration getKitConfigurationForDto(KitConfigurationDto kitConfig
             + "       krt.kit_rule_type_code as kit_rule_type,"
             + "       pe.expression_text as pex_expression,"
             + "       (select country_code from country where country_id = kc.country_id) as country_code,"
-            + "       (select group_concat(zip_code separator ',') from kit_zip_code where kit_rule_id = kr.kit_rule_id) as zip_codes"
+            + "       kz.zip_code as zip_code"
```

Comment:
I thought `group_concat` will give us all the zip codes, but apparently there's a size limit on `group_concat`, so switched it to a regular `left join`.